sunpy/map/sources/tests/test_mdi_source.py

```diff
@@ -40,6 +41,18 @@ def test_observatory(mdi):
     assert mdi.observatory == "SOHO"
 
 
+def test_reference_date(mdi):
+    assert mdi.reference_date.isot == "2010-10-15T23:00:11.000"
+
+
+def test_date(mdi):
+    assert mdi.date.isot == "2010-10-15T23:00:11.000"
+
+
+def test_private_date_setters(mdi):
+    _test_private_date_setters(mdi)
```

Comment:
Can you add another version of this test for `mdi_synoptic`?

Reply:
Will do.